src/components/Item.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+
+function Item ({id, title, price, pictureUrl, description}) {
+    return (
+            <div className="col">
+                <div className="card h-100" key={id}>
```

Comment:
A propriedade `key` precisa estar no primeiro nó, caso contrário não funcionará como esperado

Caso deseje ler sobre: https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

```
 <div className="col" key={id}>
   <div className="card h-100">
 ```